benchto-driver/src/main/java/com/teradata/benchto/driver/listeners/BenchmarkServiceExecutionListener.java

```diff
@@ -47,6 +48,8 @@
 public class BenchmarkServiceExecutionListener
```

Comment:
Admittedly, but... in fact none of the listeners could benefit it without overriding the base listener methods (not even `BenchmarkServiceExecutionListener`), so it's not actually a gain here.